enclave/core/sgx/calls.c

```diff
@@ -135,12 +136,58 @@ extern bool oe_disable_debug_malloc_check;
 **==============================================================================
 */
 
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+extern oe_eeid_t* oe_eeid;
+extern const oe_enclave_config_t* oe_enclave_config;
+
+static oe_result_t _load_config(const oe_enclave_config_t* config)
+{
+    if (config && config->data && config->size)
+    {
+        if (oe_is_within_enclave(config->data, config->size) ||
+            oe_eeid->version != OE_EEID_VERSION)
+            return OE_INVALID_PARAMETER;
+
+        /* Copy config into enclave heap */
+        oe_enclave_config_t* copy = oe_malloc(sizeof(oe_enclave_config_t));
+        if (!copy)
+            return OE_OUT_OF_MEMORY;
+        copy->size = config->size;
+        copy->data = oe_malloc(config->size);
+        if (!copy->data)
+            return OE_OUT_OF_MEMORY;
+        memcpy(copy->data, config->data, config->size);
+
+        if (!oe_is_within_enclave(copy->data, copy->size))
```

Comment:
`oe_is_within_enclave(copy->data, copy->size)` will always be true since `copy->data` is initialized via a malloc within the enclave `copy->data = oe_malloc(config->size)`